packages/blockchain-wallet-v4-frontend/src/data/coins/accountTypes/accountTypes.erc20.ts

```diff
@@ -0,0 +1,85 @@
+import { head, lift } from 'ramda'
+import { select } from 'redux-saga/effects'
+
+import { coreSelectors } from '@core'
+import { APIType } from '@core/network/api'
+import { BSBalanceType, ExtractSuccess, PaymentValue } from '@core/types'
+import { createDeepEqualSelector } from '@core/utils'
+import { selectors } from 'data'
+import { SwapAccountType, SwapBaseCounterTypes } from 'data/types'
+
+import { generateTradingAccount } from '../utils'
+import { NonCustodialAccountTypeClass } from './accountTypes.classes'
+
+export class ERC20AccountType implements NonCustodialAccountTypeClass {
+  api: APIType
+
+  networks: any
+
+  constructor(api: APIType, networks: any) {
+    this.api = api
+    this.networks = networks
```

Comment:
These are not needed in this class. There are other classes in this folder that do not need one of these.